test/fixtures/linter/rules/renderer/ControlRendererDeclaration_negative.js

```diff
@@ -19,24 +19,6 @@ sap.ui.define([
 		renderer: undefined
 	});
 
-	const NegativeExample4 = Control.extend("sap.ui.demo.linter.controls.NegativeExample4", {
-		metadata: {},
-		renderer: function(oRm, oControl) {
-		}
-	});
-
-	const NegativeExample5 = Control.extend("sap.ui.demo.linter.controls.NegativeExample5", {
-		metadata: {},
-		renderer: (oRm, oControl) => {
-		}
-	});
-
-	const NegativeExample6 = Control.extend("sap.ui.demo.linter.controls.NegativeExample6", {
-		metadata: {},
-		renderer(oRm, oControl) {
-		}
-	});
-
```

Comment:
Not related to this change:
* line 43 (new): I think the comment should read "... But no renderer _has to be_ specified." I think we don't rule out extending the renderer for a view, but it's not mandatory to define one.
* line 66 (new): The comment states "... despite that Renderers are usually not present in the ts definitions...." Not sure whether this refers to the TypeScript types, then this is correct. If it refers to TypeScript sources: